### PR TITLE
UIDEXP-39: Add shared components for second settings pane

### PR DESCRIPTION
**Pull request by YevheniiMaltsev**
## Purposes

Add shared components SettingsLabel and ProfilesLabel in scope of adding the second settings pane. [Story](https://issues.folio.org/browse/UIDEXP-39).

### CODE REVIEW

File: lib/Settings/ProfilesLabel/ProfilesLabel.css [hunk 1]
```diff
@@ -0,0 +1,13 @@
+@import "@folio/stripes-components/lib/variables.css";
+
+.profilesLabel {
+  display: flex;
+}
+.profilesLabel button {
+  margin-top: -3px
```
Comment (VictorSoroka1) [inline lib/Settings/ProfilesLabel/ProfilesLabel.css]:
```suggestion
  margin-top: -3px;
```

File: lib/Settings/SettingsLabel/SettingsLabel.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+import { AppIcon } from '@folio/stripes/core';
+
+const SettingsLabel = ({
+  messageId,
+  iconKey,
+  app,
+}) => {
+  return (
+    <div data-test-setting-label>
```
Comment (VictorSoroka1) [inline lib/Settings/SettingsLabel/SettingsLabel.js]: For consistency with component name.
```suggestion
    <div data-test-settings-label>
```

File: lib/Settings/ProfilesLabel/tests/ProfileLabel-test.js [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+
+      it('should display put content', () => {
+        expect(popover.content.isPresent).to.be.true;
+      });
+
+      it('should apply correct href property', () => {
+        expect(popover.linkHref).to.equal(link);
+      });
+
+      it('should display put content', () => {
```
Comment (VictorSoroka1) [inline lib/Settings/ProfilesLabel/tests/ProfileLabel-test.js]: Let's name for better clarity. Same [here](https://github.com/folio-org/stripes-data-transfer-components/pull/23/files#diff-af8755bd52a3d965735846c6f4fefd1cR63).
```suggestion
      it('should display passed in content', () => {
```

File: lib/Settings/ProfilesLabel/tests/ProfileLabel-test.js [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+    });
+
+    it('should display info icon button', () => {
+      expect(profilesLabel.infoButton.isPresent).to.be.true;
+    });
+
+    describe('clicking on info icon button', () => {
+      beforeEach(async () => {
+        await profilesLabel.infoButton.click();
+        await new Promise(resolve => { setTimeout(() => resolve(), 1000); });
```
Comment (VictorSoroka1) [inline lib/Settings/ProfilesLabel/tests/ProfileLabel-test.js]: I think in the long run it makes sense to move this to utils with the ability to pass timer value as a parameter.

File: lib/Settings/ProfilesLabel/tests/ProfileLabel-test.js [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+          content={content}
+        />
+      );
+    });
+
+    afterEach(() => {
+      document.getElementById('OverlayContainer').remove();
+    });
+
+    it('should display ProfileLabel component', () => {
```
Comment (VictorSoroka1) [inline lib/Settings/ProfilesLabel/tests/ProfileLabel-test.js]: I suggest to not bound to component names as they may change over time.
```suggestion
    it('should display profile label', () => {
```

File: lib/Settings/ProfilesLabel/tests/ProfileLabel-test.js [hunk 6]
```diff
@@ -0,0 +1,76 @@
...
+import { mountWithContext } from '../../../../test/bigtest/helpers';
+import {
+  ProfilesLabelInteractor,
+  ProfilesPopoverInteractor,
+} from './interactors';
+import ProfilesLabel from '../ProfilesLabel';
+
+import translations from '../../../../translations/stripes-data-transfer-components/en';
+
+describe('ProfilesLabel', () => {
```
Comment (VictorSoroka1) [inline lib/Settings/ProfilesLabel/tests/ProfileLabel-test.js]: Should we have `rendering` part for consistency? I see that we have the wording like that for the majority of components.
```suggestion
describe('rendering ProfilesLabel', () => {
```

Comment (YevheniiMaltsev) [reply lib/Settings/ProfilesLabel/tests/ProfileLabel-test.js]: It is consistent with other modules. Please, take a look a the line 23 and other components

File: lib/Settings/ProfilesLabel/tests/ProfileLabel-test.js (hunk 4 above)
Comment (YevheniiMaltsev) [reply lib/Settings/ProfilesLabel/tests/ProfileLabel-test.js]: Yes, you are right. Let's do it when we have other places with timeout.

File: lib/Settings/ProfilesLabel/tests/ProfileLabel-test.js (hunk 6 above)
Comment (VictorSoroka1) [reply lib/Settings/ProfilesLabel/tests/ProfileLabel-test.js]: You are right.

**Pull request merged**